addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/HiddenFilesScanRule.java

```diff
@@ -69,7 +69,8 @@ public class HiddenFilesScanRule extends AbstractHostPlugin {
             CommonAlertTag.toMap(
                     CommonAlertTag.OWASP_2021_A05_SEC_MISCONFIG,
                     CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG,
-                    CommonAlertTag.WSTG_V42_CONF_05_ENUMERATE_INFRASTRUCTURE);
+                    CommonAlertTag.WSTG_V42_CONF_05_ENUMERATE_INFRASTRUCTURE,
+                    CommonAlertTag.CUSTOM_PAYLOADS);
```

Comment:
IMHO this should be added only to example alerts, if I saw this in regular alerts I'd think the alert was raised because of custom payloads not that the scan rule supports them.

Reply:
Okay, I’ll look at reworking this.